yt/visualization/volume_rendering/tests/test_varia.py

```diff
@@ -63,11 +63,8 @@ def test_multiple_fields(self):
     def test_rotation_volume_rendering(self):
         im, sc = yt.volume_render(self.ds)
 
-        angle = 2 * np.pi
-        frames = 4
-        for i in range(frames):
-            sc.camera.yaw(angle / frames)
-            sc.render()
+        sc.camera.yaw(np.pi)
```

Comment:
This one I'm sort of -0 on removing.

Reply:
Do you think it helps test something that isn't in other tests? Or?

Reply:
Nah, it's probably fine. I'm just a bit reactionary.